src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -599,7 +549,6 @@ impl<'a, 'tcx> Visitor<'tcx> for Qualifier<'a, 'tcx, 'tcx> {
             Operand::Move(_) => {
                 self.nest(|this| {
                     this.super_operand(operand, location);
-                    this.try_consume();
```

Comment:
You can remove the `self.nest` around the `super_operand` call - its only purpose was to observe *only* the flags being added by `super_operand`, in `try_consume`.

Reply:
In fact, you can put `self.super_operand(operand, location)` at the top of the function, before the `match`.

Reply:
Got it.